### PR TITLE
[UP-422] Example medication schedule document

### PR DESCRIPTION
**Pull request by kellemNegasi**
creates medication schedule document according to [UP-422](https://unidoc.atlassian.net/browse/UP-422)

### CODE REVIEW

**Review by 3ace — CHANGES_REQUESTED** [review]

**Review by 3ace — APPROVED** [review]
lgtm

**Review by sampila — APPROVED** [review]
Looks good to me

File: templates/medication-schedule-report/templates/main.tpl [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+                <text-chunk font="arial-bold" font-size="11">{{.Name}}</text-chunk>
+            </paragraph>
+            <paragraph margin="5 5 5 0">
+                <text-chunk font="arial" font-size="9">{{.Description}}</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+    {{$daysTaken := .DaysTaken}}
+    {{range .TimesOfTheDay}}
+        <table-cell border-width-right="0.5" border-width-bottom="0.5" vertical-align="bottom" align="left" indent="0">
```
Comment (adrg) [inline templates/medication-schedule-report/templates/main.tpl:34]: No need to specify the `align` attribute as `left` is the default.

File: templates/medication-schedule-report/templates/main.tpl [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        <table-cell border-width-bottom="1.5" vertical-align="bottom">
+            <paragraph margin="0 0 10 0">
+                <text-chunk font="arial-bold">{{.}}</text-chunk>
+            </paragraph>
+        </table-cell>
+    {{end}}
+{{end}}
+
+{{define "drug-schedule"}}   
+    <table-cell border-width-left="0.5" border-width-bottom="0.5" vertical-align="top" rowspan="{{len .TimesOfTheDay}}">
```
Comment (adrg) [inline templates/medication-schedule-report/templates/main.tpl:22]: No need to specify the `vertical-align` attribute as `top` is the default.

File: templates/medication-schedule-report/templates/main.tpl [hunk 3]
```diff
@@ -0,0 +1,207 @@
+{{define "table-header"}}
+    <table-cell  border-width-bottom="1.5" vertical-align="bottom">
+        <paragraph margin="0 0 10 0">
+            <text-chunk font="arial-bold">Drug &amp; Usage</text-chunk>
+        </paragraph>
+    </table-cell>
+    <table-cell  border-width-bottom="1.5" vertical-align="bottom" align="left" indent="0">
```
Comment (adrg) [inline templates/medication-schedule-report/templates/main.tpl:7]: No need to specify the `align` attribute as `left` is the default.

File: templates/medication-schedule-report/templates/main.tpl [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    </table-cell>
+    <table-cell vertical-align="{{.Valign}}" border-width-bottom="{{.BorderWidth}}" indent="0">
+        <paragraph>
+            <text-chunk font="arial" font-size="{{.FontSize}}">{{.Col3}}</text-chunk>
+        </paragraph>
+    </table-cell>
+{{end}}
+
+<table columns="5" indent="0" column-widths="0.15 0.55 0.03 0.2 0.07">
+    <table-cell vertical-align="top">
```
Comment (adrg) [inline templates/medication-schedule-report/templates/main.tpl:119]: No need to specify the `vertical-align` attribute as `top` is the default.

File: templates/medication-schedule-report/pdf_medication_schedule.go [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+	}
+
+	// Create template options.
+	tplOpts := &creator.TemplateOptions{
+		FontMap: map[string]*model.PdfFont{
+			"arial-bold": arialBold,
+			"arial":      arial,
+		},
+		HelperFuncMap: template.FuncMap{
+			"getColumnWidths": func(NumOfCols int, colWidth float64) string {
```
Comment (adrg) [inline templates/medication-schedule-report/pdf_medication_schedule.go:85]: Please start variable names with lowercase letters: `NumOfCols` -> `numOfCols`.

**Review by adrg — COMMENTED** [review]
Looks good. I left a couple of minor observations.

**Comment by kellemNegasi** [comment]
Thanks @adrg I have updated the PR according to your comments.

**Review by adrg — APPROVED** [review]
Looks good. Thank you for addressing the review points.

**Review by sampila — APPROVED** [review]
looks good to me

**Pull request merged**